content/python/concepts/strings/terms/isascii/isascii.md

```diff
@@ -0,0 +1,53 @@
+---
+Title: '.isascii()'
+Description: 'Returns True if all characters in a string are ASCII characters; otherwise, it returns False.'
+Subjects:
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Methods'
+  - 'Functions'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
```

Comment:
```suggestion
```

Reply:
our standard is to list 2 items only under catalog content

Reply:
Got it